modules/init.gradle

```diff
@@ -6,6 +6,14 @@ import org.gradle.internal.hash.HashValue
 if (System.properties["http.proxyHost"] == "squid.lax.liferay.com") {
 	logger.lifecycle "Using proxy {}:{}.", System.properties["http.proxyHost"], System.properties["http.proxyPort"]
 
+	File repositoryLocalDir = file("../.m2-tmp")
+
+	URI uri = repositoryLocalDir.toURI()
+
+	String repositoryLocalUrl = uri.toASCIIString()
```

Comment:
No need to create this variable, use `uri.toASCIIString()` directly when setting the system property.